pset0/satyamvats5/marksheet/1.py

```diff
@@ -0,0 +1,18 @@
+class student:
+	def __init__(self, name, roll, m1, m2, m3, m4, m5):
+		self.name = name
+		self.roll = roll
+		self.m = [m1, m2, m3, m4, m5]
+students = []
+num = int(input())
+for i in range(num):
+	name, roll, m1, m2, m3, m4, m5 = input().split(',')
+	students.append(student(name, roll, m1, m2, m3, m4, m5))
+tmp = int(input())
+for j in range(tmp):
+	r, sub = input().split(',')
+	r = int(r)
+	sub = int(sub)	
+	for i in students:
+		if (int(i.roll) == r):
+			print("{} {}".format(i.name,i.m[sub-1])) 
```

Comment:
A single space after the comma.
Also, [add a newline at the end of the file](https://stackoverflow.com/a/44317793).